### PR TITLE
add suffix for ChunkEvaluator

### PR DESCRIPTION
**Pull request by linjieccc**
add suffix for token-cls task to support different type of labels

### CODE REVIEW

File: modules/text/language_model/chinese_electra_base/README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ def __init__(
         load_checkpoint=None,
         label_map=None,
         num_classes=2,
-        **kwargs,
+    suffix=False,
+    **kwargs,
 )
```
Comment (KPatr1ck) [inline modules/text/language_model/chinese_electra_base/README.md:20]:
```python
        num_classes=2,
        **kwargs,
    suffix=False,
    **kwargs,
```

参数没有对齐

File: modules/text/language_model/chinese_bert_wwm_ext/module.py [hunk 2]
```diff
@@ -70,7 +71,7 @@ def __init__(
             self.model = BertForTokenClassification.from_pretrained(
                 pretrained_model_name_or_path='bert-wwm-ext-chinese', num_classes=self.num_classes, **kwargs)
             self.criterion = paddle.nn.loss.CrossEntropyLoss()
-            self.metric = ChunkEvaluator(label_list=[self.label_map[i] for i in sorted(self.label_map.keys())])
+            self.metric = ChunkEvaluator(label_list=[self.label_map[i] for i in sorted(self.label_map.keys())], suffix)
```
Comment (KPatr1ck) [inline modules/text/language_model/chinese_bert_wwm_ext/module.py]: 建议保持统一用kwarg传递suffix，方便维护

File: modules/text/language_model/chinese_electra_base/README.md (hunk 1 above)
Comment (linjieccc) [reply modules/text/language_model/chinese_electra_base/README.md:20]: fixed

File: modules/text/language_model/chinese_bert_wwm_ext/module.py (hunk 2 above)
Comment (linjieccc) [reply modules/text/language_model/chinese_bert_wwm_ext/module.py]: done

**Pull request merged**